app/controllers/ChargeSummaryController.scala

```diff
@@ -131,8 +131,7 @@ class ChargeSummaryController @Inject()(val auth: AuthenticatorPredicate,
     val chargeItem = ChargeItem.fromDocumentPair(
       documentDetailWithDueDate.documentDetail,
       financialDetailsForCharge,
-      codingOut = true,
-      reviewAndReconcile = isEnabledFromConfig(ReviewAndReconcilePoa))
+      codingOut = true)
```

Comment:
**codingOut** flag is not in use in fromDocumentPair method, should be ok to drop it;